flocker/common/script.py

```diff
@@ -70,6 +71,42 @@ def main(reactor, options):
         :return: A ``Deferred`` which fires when the script has completed.
         """
 
+# This should probably be built-in functionality in Eliot;
+# see https://github.com/ClusterHQ/eliot/issues/143
```

Comment:
Adding a link to https://github.com/ClusterHQ/eliot/issues/143 so that we can see the code that issue refers to.